Scripts/Miscellaneous/Streamlit_webapp/README.md

```diff
@@ -0,0 +1,20 @@
+# Streamlit Webapp
+
+This is a python script for creating a simple compound interest calculator webapp.
+# Pre-requisites:
+
+Install the required modules:
+```bash
+pip install -r requirements.txt
+```
+# Instructions to run this script:
+
+```bash
+python Compound_interest.py
```

Comment:
Stremlit apps are not executed like this

it should be streamlit run Compound_interest.py